dashboard/app/models/school_district.rb

```diff
@@ -0,0 +1,48 @@
+# == Schema Information
+#
+# Table name: school_districts
+#
+#  id         :integer          not null, primary key
+#  name       :string(255)
+#  city       :string(255)
+#  state      :string(255)
+#  zip        :string(255)
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+class SchoolDistrict < ActiveRecord::Base
+  include Seeded
+
+  CSV_HEADERS =
+  {
+    :id => 'LEAID',
+    :name => 'NAME',
+    :city => 'LCITY',
+    :state => 'LSTATE',
+    :zip => 'LZIP'
+  }
+
+  # Use the zero byte as the quote character to allow importing double quotes
+  #   via http://stackoverflow.com/questions/8073920/importing-csv-quoting-error-is-driving-me-nuts
+  CSV_IMPORT_OPTIONS = { col_sep: "\t", headers: true, :quote_char => "\x00" }
+
+  def self.find_or_create_all_from_tsv!(filename)
+    created = []
+    CSV.read(filename, CSV_IMPORT_OPTIONS).each do |row|
+      created << self.first_or_create_from_tsv_row!(row)
```

Comment:
Nit: prefer `map`.  Can this code be de-duped against the similar code in [`callout.rb`](https://github.com/code-dot-org/code-dot-org/blob/a7800a69e3153f7bb8d7bb32fe3e9c13f9170f40/dashboard/app/models/callout.rb#L37)?